src/main/java/org/assertj/core/condition/MappedCondition.java

```diff
@@ -133,11 +133,19 @@ public boolean matches(FROM value) {
   protected String buildMappingDescription(FROM from, TO to) {
     StringBuilder sb = new StringBuilder("mapped");
     if (!mappingDescription.isEmpty()) sb.append(format("%n   using: %s", mappingDescription));
-    sb.append(format("%n   from: <%s> %s%n", from.getClass().getSimpleName(), from));
-    sb.append(format("   to:   <%s> %s%n", to.getClass().getSimpleName(), from, to));
+    sb.append(format("%n   from: <%s> %s%n", classNameOrNull(from), from));
+    sb.append(format("   to:   <%s> %s%n", classNameOrNull(to), from, to));
     sb.append("   then checked:");
     sb.append(format("%n      %-10s", condition));
     return sb.toString();
   }
 
+  private static String classNameOrNull(Object object) {
+
+    if (object == null) {
+      return "<null>";
+    }
+    return object.getClass().getSimpleName();
```

Comment:
use ternary operator `?` to get a one liner for `classNameOrNull`

Reply:
changed